CMakeLists.txt

```diff
@@ -65,8 +65,14 @@ add_subdirectory(test)
 
 install(PROGRAMS ${CLANG_OCL} DESTINATION bin)
 
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "rocm-llvm, rocm-opencl-dev")
-set(CPACK_RPM_PACKAGE_REQUIRES "rocm-llvm, rocm-opencl-devel")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "rocm-llvm, rocm-opencl-dev, rocm-core")
+set(CPACK_RPM_PACKAGE_REQUIRES "rocm-llvm, rocm-opencl-devel, rocm-core")
+# Remove dependency on rocm-core if -DROCM_DEP_ROCMCORE=ON not given to cmake
```

Comment:
Why do we need a flag to disable this?

Reply:
Because "requirements"! The argument is that rocm-core has not yet been open sourced, so people who don't have access to the internal source trees will be stopped from building. This is also the reason why the dependency defaults to OFF, so there is no change to their existing workflows.